app/services/send_chase_email_to_provider.rb

```diff
@@ -0,0 +1,5 @@
+class SendChaseEmailToProvider
+  def self.call(application_choice:)
+    ChaserSent.create!(chased: application_choice.provider, chaser_type: :provider_decision_request)
```

Comment:
This is slightly off I think! We're saving one ChaserSent per `provider`, so this would cause only one `provider_decision_request` to be sent per provider ever. What we want to achieve is that we send only one email to the provider per application choice.